02_getting_started.asciidoc

```diff
@@ -189,44 +189,44 @@ Both Bitcoin addresses and QR codes contain error detection information that pre
 
 [[eclair-tx1]]
 .Alice receives bitcoin
 image:images/eclair-tx1-btc.png[]
 
-While in this example Alice used an ATM to acquire her first bitcoin, the same basic concepts would apply even if she used one of the other methods in <<acquiring-bitcoin>>. For example, if Alice wanted to sell a product or provide a professional service in exchange for bitcoin, her customers could scan the Bitcoin Address with their own wallets and pay her in bitcoin.
+While in this example Alice used an ATM to acquire her first bitcoin, the same basic concepts would apply even if she used one of the other methods in <<acquiring-bitcoin>>. For example, if Alice wanted to sell a product or provide a professional service in exchange for bitcoin, her customers could scan the Bitcoin Address with their wallets and pay her in bitcoin.
 
-Similarly, if she billed a client for a service offered over the Internet, Alice could send an email or instant message with the Bitcoin Address or the QR code (or both) to her client and they could scan or paste the information into a Bitcoin wallet to pay her.
+Similarly, if she billed a client for a service offered over the Internet, Alice could send an email or instant message with the Bitcoin Address or the QR code to her client, and they could paste or scan the information into a Bitcoin wallet to pay her.
 
-Alice could even print the QR code and affix it to a sign and display it to receive tips. A singer who receives bitcoin tips has a QR code printed and affixed to their guitar while performing!
+Alice could even print the QR code and affix it to a sign and display it to receive tips. For example, she could have a QR code affixed to her guitar and receive tips while performing!
 
 Finally, if Alice bought bitcoin from a crypto-currency exchange, she could (and should!) then "withdraw" the bitcoin by pasting her Bitcoin Address into the exchange website. The exchange will then send the bitcoin to her address directly.
 
 === From Bitcoin to Lightning Network
 
-Alice's bitcoin is now controlled by her Eclair wallet and has been recorded on the Bitcoin Blockchain. At this point, Alice's bitcoin is "on-chain", meaning that she has completed a regular Bitcoin transaction, broadcast to the entire Bitcoin network, verified by all Bitcoin nodes and "mined" (recorded) onto the Bitcoin blockchain.
+Alice's bitcoin is now controlled by her Eclair wallet and has been recorded on the Bitcoin Blockchain. At this point, Alice's bitcoin is "on-chain," meaning that the transaction has been broadcast to the entire Bitcoin network, verified by all Bitcoin nodes, and "mined" (recorded) onto the Bitcoin blockchain.
```

Comment:
`"on-chain,"` why is the comma inside the quotation marks?

Reply:
It's how it's done in [American English vs British English](https://www.thepunctuationguide.com/british-versus-american-style.html). And Andreas confirmed that the book would be using American English. 

To me it also looks strange to have punctuation go inside quotation marks, even when it's not part of the quote, but it's standard in America.